libs/wingc/src/type_check.rs

```diff
@@ -5599,6 +5604,29 @@ impl<'a> TypeChecker<'a> {
 			.map(|_| base_udt)
 	}
 
+	fn make_immutable(&mut self, type_: TypeRef) -> TypeRef {
+		match *type_ {
+			Type::MutArray(inner) => {
```

Comment:
what about `MutJson`?

Reply:
What about it? It's handled 4 lines down. Am I missing something?

Reply:
No, I've missed it. Sorry!